build/app.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
```

Comment:
Why do you generate all these `build` files? You must setup your app the way not to generate these files unless you use Typescript